### PR TITLE
switch kuscia to bzlmod

### PR DESCRIPTION
**Pull request by fucangfy**

### CODE REVIEW

**Comment by fucangfy** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .bazelrc [hunk 1]
```diff
@@ -18,6 +18,10 @@ build --incompatible_new_actions_api=false
 build --copt=-fdiagnostics-color=always
 build --enable_platform_specific_config
 
+common --registry=https://raw.githubusercontent.com/secretflow/bazel-registry/main
+common --registry=https://bcr.bazel.build
+common --registry=https://baidu.github.io/babylon/registry
```
Comment (tongke6) [inline .bazelrc:23]: kuscia only depends on brc ?

Comment (fucangfy) [reply .bazelrc]: done

**Review by tongke6 — APPROVED** [review]
LGTM

@UniqueMarvin please take a final look

**Review by UniqueMarvin — APPROVED** [review]
LGTM

**Pull request merged**